core/src/main/java/com/novoda/noplayer/internal/exoplayer/CompositeTrackSelectorCreator.java

```diff
@@ -14,16 +13,13 @@
 
 class CompositeTrackSelectorCreator {
 
-    private final DefaultBandwidthMeter bandwidthMeter;
-
-    CompositeTrackSelectorCreator(DefaultBandwidthMeter bandwidthMeter) {
-        this.bandwidthMeter = bandwidthMeter;
-    }
-
     CompositeTrackSelector create(Options options) {
+        DefaultBandwidthMeter bandwidthMeter = new DefaultBandwidthMeter.Builder()
```

Comment:
Should be good to have this separate from the other one from what I have seen. 🏈

Reply:
Whilst we have more control over rendition selecting across sources it also means we have multiple places to change. 

How  do you feel about introduce a holder for audio/video/text bandwidth meters? and potentially be default we could reuse the same one

Reply:
I opted to pass through in the two places it is used, they both get called in the same place on the Facade 😄 also helpful to ensure that it is the same instance, it might have some affect that I haven't accounted for 😄

Reply:
:100: nice! I'm not sure about two independent instances yet, it might be safer to test first but with a single place to pass them from I'm happy